lib/galaxy/managers/dbkeys.py

```diff
@@ -17,9 +21,9 @@
 
 def read_dbnames(filename):
     """Read build names from file"""
-    db_names = []
+    db_names: List = []
```

Comment:
Not necessary, unless you have to specify the list content type.

Reply:
```
(.venv) rivendell$ tox -e mypy
mypy: commands[0]> mypy test lib
lib/galaxy/managers/dbkeys.py:24: error: Need type annotation for "db_names" (hint: "db_names: List[<type>] = ...")  [var-annotated]
        db_names = []
        ^~~~~~~~
```

Reply:
These were due to:

```ini
[mypy-galaxy.util.dbkeys]
check_untyped_defs = False
```

in `mypy.ini` which we can now be removed.

Reply:
I've tried - same error as before. (I've cleared the cache) Does it work for you?

Reply:
I'm updating mypy.ini anyway, of course.

Reply:
Thanks, I've improved the type annotation (and an import) in 2 commits I've just pushed to your branch, hope that's OK.

Reply:
of course, thanks!